lib/mix/lib/mix/generator.ex

```diff
@@ -71,7 +71,7 @@ defmodule Mix.Generator do
 
   ## Examples
 
-      iex> Mix.Generator.copy_file("source/gitignore", ".gitignore", "_build\ndeps\n")
```

Comment:
third argument should be a keyword list of options, this appears to be a copy/paste error from the `create_file` docs